modules/smart-agent_cassandra-nodetool/conf/readme.yaml

```diff
@@ -0,0 +1,3 @@
+documentations:
+
+source_doc:
```

Comment:
you should inform it requires to use the groovy script, probably add this script to git versionning in this module directory and ideally provide a sample agent config